openapi-to-har.js

```diff
@@ -179,6 +179,45 @@ const getParameterValues = function (param, values) {
   };
 };
 
+/**
+ * Parse parameter object into query string objects
+ *
+ * @param {Object} openApi    OpenApi document
+ * @param {Object} parameters Objects described in the document to parse into the query string
+ * @param  {Object} values    Optional: query parameter values to use in the snippet if present
+ * @return {array}            List of objects describing the query strings
+*/
+const parseParametersToQuery = function (openApi, parameters, values) {
+  const queryStrings = [];
+
+  for (let i in parameters) {
+    let param = parameters[i];
+    if (typeof param['$ref'] === 'string' && /^#/.test(param['$ref'])) {
```

Comment:
@bpross This code looks a little strange. Not sure what you are trying to do but what happens if there are 3 levels of nesting?